purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/PaywallAnimation.kt

```diff
@@ -43,8 +34,14 @@ class PaywallAnimation(
 }
 
 @OptIn(InternalRevenueCatAPI::class)
-object AnimationTypeAsObjectSerializer : EnumAsObjectSerializer<PaywallAnimation.AnimationType>(
```

Comment:
Do we need to delete `EnumAsObjectSerializer` as well? I believe it's unused now?